widget_driver_generator/test/builder_test.dart

```diff
@@ -1,4 +1,4 @@
-import 'package:flutter_test/flutter_test.dart';
+import 'package:test/test.dart';
```

Comment:
Did you also test that the script which runs the test still works? I think I had to make change in the annotation-test inn that script to make it work after I changed it to use dart-test instead of flutter-test